modules/sitemap/sitemap.ts

```diff
@@ -0,0 +1,22 @@
+import SitemapRenderer from '@storefront-x/sitemap/SitemapRenderer.js'
+import modules from '~/.sfx/sitemap'
+import locales from '~/.sfx/sitemap/locales'
+
+export default async (baseUrl: string, hostname: string) => {
+  let store = null
+  for (const locale of Object.values(locales)) {
+    store = locale(baseUrl)
```

Comment:
This is nonsense. You will always override last value.

Reply:
If you had something like transformators it could look like this


```
for (const transformator of Object.values(transformators)) {
  store = transformator(store, baseUrl)
```